index.js

```diff
@@ -104,25 +120,21 @@ module.exports = {
             'done': function (filename, done, err) {
                 this.snap(filename).
                     then(function (imageObject) {
-                        var output = (imageObject.imageUrl) ?
-                        '\nnemo-screenshot\n' + imageObject.imageUrl + '\n' :
-                        '\nnemo-screenshot::' + JSON.stringify(imageObject) + '::nemo-screenshot';
-                        if (err) {
-                            err.stack = err.stack + output;
-                        }
+                        appendImageUrlToErrorStack(imageObject, err);
                         done(err);
                     }, function (scerror) {
                         console.log('nemo-screenshot encountered some error.', scerror.toString());
                         done(scerror);
                     });
+            },
+
+            'setCurrentTestTitle': function(title) {
+                this._currentTestTitle = title;
             }
```

Comment:
Oh yeah. Great point! I didn't think about the fact that this modified the public API. @jugend can you please add a section to describe this new feature?